.npmrc

```diff
@@ -0,0 +1 @@
+prefix=/Users/anhdungle/.nvm/versions/node/v6.11.1
```

Comment:
Looks like this belongs into the .gitignore because it has a path from your computer.

Reply:
fix .gitignore